tests/components/islamic_prayer_times/test_init.py

```diff
@@ -145,3 +147,46 @@ async def test_update(hass: HomeAssistant) -> None:
         async_fire_time_changed(hass, future)
         await hass.async_block_till_done()
         assert pt_data.data == NEW_PRAYER_TIMES_TIMESTAMPS
+
+
+@pytest.mark.parametrize(
+    ("object_id", "old_unique_id"),
+    [
+        (
+            "fajer_prayer",
+            "Fajr",
+        ),
+        (
+            "dhuhr_prayer",
+            "Dhuhr",
+        ),
+    ],
+)
+async def test_migrate_unique_id(
+    hass: HomeAssistant, object_id: str, old_unique_id: str
+) -> None:
+    """Test unique id migration."""
+    entry = MockConfigEntry(domain=islamic_prayer_times.DOMAIN, data={})
+    entry.add_to_hass(hass)
+
+    ent_reg = er.async_get(hass)
+
+    entity: er.RegistryEntry = ent_reg.async_get_or_create(
```

Comment:
Explicit typing here isn't needed.

Reply:
Please only use `entity` variable names for `Entity` instances. Call this eg `entity_entry`.